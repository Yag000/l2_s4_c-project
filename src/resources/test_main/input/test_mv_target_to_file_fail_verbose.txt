touch objective
mkdir k
mv k objective/l
