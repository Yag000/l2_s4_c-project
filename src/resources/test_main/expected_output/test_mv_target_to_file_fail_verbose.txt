Parsing file src/resources/test_main/input/test_mv_target_to_file_fail_verbose.txt ...
/$ touch objective
/$ mkdir k
/$ mv k objective/l
Invalid path for the movement.
